fieldservice_mobile/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     "category": "Field Service",
     "author": "Open Source Integrators",
     "website": "https://github.com/ursais/osi-addons",
-    "depends": ["fieldservice_stage_server_action", "stock"],
+    "depends": ["fieldservice_stage_server_action", "stock", "fieldservice_stock"],
```

Comment:
@ReedHayashikawa Please Remove fieldservice_stock as a dependency because `stock.move.line`, `stock.location` and `stock.warehouse` models are created in **stock** module

Reply:
Removed fieldservice_stock @murtuzasaleh